plugins/pur/core/src/main/java/org/pentaho/di/repository/pur/UserRoleDelegate.java

```diff
@@ -95,8 +102,18 @@ private void initManaged( PurRepositoryMeta repositoryMeta, IUser userInfo ) thr
     HTTPBasicAuthFilter authFilter = new HTTPBasicAuthFilter( userInfo.getLogin(), userInfo.getPassword() );
     Client client = new Client();
     client.addFilter( authFilter );
-    WebResource resource = client.resource( webService );
-    String response = resource.accept( MediaType.APPLICATION_JSON_TYPE ).get( String.class );
+
+    WebResource.Builder resource = client.resource( webService ).accept( MediaType.APPLICATION_JSON_TYPE );
+    /**
+     * if set, _trust_user_ needs to be considered. See other places in pur-plugin's:
+     *
+     * @link https://github.com/pentaho/pentaho-kettle/blob/8.0.0.0-R/plugins/pur/core/src/main/java/org/pentaho/di/repository/pur/PurRepositoryConnector.java#L97-L101
+     * @link https://github.com/pentaho/pentaho-kettle/blob/8.0.0.0-R/plugins/pur/core/src/main/java/org/pentaho/di/repository/pur/WebServiceManager.java#L130-L133
+     */
+    if ( StringUtils.isNotBlank( System.getProperty( "pentaho.repository.client.attemptTrust" ) ) ) {
```

Comment:
Do we start pan/kitchen with this `pentaho.repository.client.attemptTrust` env variable?

Reply:
See, for example, https://github.com/pentaho/pentaho-kettle/blob/8.0.0.0-R/plugins/pur/core/src/main/java/org/pentaho/di/repository/pur/WebServiceManager.java#L130-L133